### DISCUSSION TITLE
wordpress equivalent

### DISCUSSION THREAD

**Discussion by jpjoneshy1** [e107inc/e107#5047] [Q&A Core]
Hello, new to the e107 discussion group.
Wordpress has a file I'd like to know what the equivalent in e107 is:  wp-blog-header.php
Thanks,
JP

**Comment by Jimmi08** [comment]
Hi,
that file is used to load WP environment, e107 alternative is class2.php

**Reply by jpjoneshy1** [reply]
Thank you Jimmi08.  I had used e107 for years, up until they changed it to the bootstrap.  Just couldn't wrap my head around it at the time.  I've been using WordPress and the genealogy app TNG now for a while, but still only have a partial website built.  So I wanted try an experiment with e107 and TNG.  Before the Bootstrap we had a plugin for that.  I'm not looking for a 'login integration to both apps', although that would be nice.
Thanks again,
JP

**Comment by Jimmi08** [comment]
I use wp and e107 installed in subdirectory. I wrote wp shortcode that use
e107 stuff - like db connect, tablerender and I use it in wp page.
I am not able to get used to wp way of working with data. Just - it is long
time ago and nobody of real developers was interested of this direction.

There is older article about this on my site, I am on way for holiday now,
so if you are interested in this topic, I will be back on August.

**Comment by jpjoneshy1** [comment]
Interesting.  Yes I would be interested in that article when you return from holiday.  Have a safe journey.

 

Thanks,

JP

**Reply by Jimmi08** [reply]
Hi,
ok, here is the article: [https://www.e107sk.com/news/view/107/combination-of-wordpress-and-e107](https://www.e107sk.com/news/view/107/combination-of-wordpress-and-e107) 

TNG is not free software so I can't look at it but I would go for e107 combination, without WP stuff. 
Now I am busy with my project of combination e107 and efiction script/cms. Until I finish it to some standard level, I don't do anything else.

By the way, there was e107 1.0.4 updated for PHP 7 but it is the short-time solution.
